alerts/service.libsonnet

```diff
@@ -0,0 +1,28 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'service.rules',
+        rules: [
+          {
+            alert: 'AbsentRookCephMgr',
```

Comment:
Please make this alert disabled by default , as Rook is not standard way to install Ceph currently 

As per official docs http://docs.ceph.com/docs/master/dev/kubernetes/?highlight=rook#hacking-on-ceph-in-kubernetes-with-rook
```
Warning
This is not official user documentation for setting up production Ceph clusters with Kubernetes. It is aimed at developers who want to hack on Ceph in Kubernetes. 
```

And this alert wouldn't work for current stantard - ansible & ceph-deploy tools

Reply:
Also how its different from /pull/50?

Reply:
@devopsjonas  I have taken care of making the alert optional. Please review

Reply:
> Also how its different from [/pull/50](https://github.com/ceph/ceph-mixins/pull/50)?

@shtripat This alert is different as this is targeting  specific service deployed in rook.